Source/TAS/Manager.cs

```diff
@@ -11,6 +11,12 @@ internal class DisableRunAttribute : Attribute;
 
 public static class Manager
 {
+    public static World? world;
+    public static Vec3 FreeCamPosition;
+    public static Vec2 FreeCamRotation;
+    public static float FreeCamDistance = 50f;
```

Comment:
I don't like there being to placed for the freecam data, so could those and the ones in player be merged into 1?
Also if you do that, move the constants next to the fields.